### PR TITLE
FanClub and FanClubMember models created

### PR DESCRIPTION
**Pull request by rado196**
1) Database models was created with their relations.
2) Enums for Fan club model created.

### CODE REVIEW

File: packages/utils/src/enums/FanClubMemberRoleEnum.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  /**
+   * FanClub moderator.
+   */
+  moderator = 'moderator',
+
+  /**
+   * FanClub standard.
+   */
+  standard = 'standard',
```
[the remark this answers is not in the document]
Comment (rado196) [reply packages/utils/src/enums/FanClubMemberRoleEnum.ts:15]: `standard` means standard member, may I need to write just `member` instead?

**Review by rubenaprikyan — APPROVED** [review]
Lgtm!

File: apis/core-api/src/core/data/models/FanClub.ts [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  public id!: DbIdentifier;
+  public name!: string;
+  public description!: string | null;
+  public countryId!: DbIdentifier;
+  public cityId!: DbIdentifier;
+  public teamId!: DbIdentifier;
+  public avatar!: string;
+  public coverPhoto!: string | null;
+  public official!: boolean;
+  public clubPrivacy!: FanClubPrivacyEnum;
```
[the remark this answers is not in the document]
Comment (rado196) [reply apis/core-api/src/core/data/models/FanClub.ts]: updated

File: packages/utils/src/enums/FanClubMemberRoleEnum.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (rado196) [reply packages/utils/src/enums/FanClubMemberRoleEnum.ts]: changed to member

**Pull request merged**